### ISSUE TITLE
Example is broken?

### ISSUE THREAD

**Issue by pixelmund** [AlecAivazis/houdini#133]
Hey, i just pulled a fresh copy of houdini and running:

- yarn && yarn build
- cd example -> yarn api 
- cd example -> yarn generate && yarn dev

It's throwing following error 

```bash
Failed to resolve import "./ItemEntry_item" from "$houdini/artifacts/index.js". Does the file exist?
```

If i look into the `artifacts` directory indeed there is no `ItemEntry_item.js` file. Only the `ItemEntry_item.d.ts` file is generated, all other artifacts are getting generated properly.

**Comment by AlecAivazis** [comment]
Huh thanks for reporting this @pixelmund - i verified it on my end. When I test the example app I run  generate with everything in place so I missed that this happened. Will get a fix out very shortly

**Comment by AlecAivazis** [comment]
This should be fixed once #134 goes in
